src/pymor/reductors/aaa.py

```diff
@@ -0,0 +1,391 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+import itertools
+
+import numpy as np
+
+from pymor.core.base import BasicObject
+from pymor.models.transfer_function import TransferFunction
+
+
+class pAAAReductor(BasicObject):
+    """Reductor implementing the parametric AAA algorithm.
+
+    The reductor implements the parametric AAA algorithm and can be used either with
+    data or a given full-order model which has a `transfer_function` attribute. MIMO
+    data is accepted.
+
+    Parameters
+    ----------
+    sampling_values
+        Values where sample data has been evaluated or the full-order model should be evaluated.
+        Sampling values are represented as a nested list `svs` such that `svs[i]`corresponds
+        to sampling values of the `i-th` variable.
```

Comment:
It should be mentioned which coordinate corresponds to `s` for transfer functions and models with transfer functions.